client/overview.html

```diff
@@ -0,0 +1,74 @@
+<!-- <!DOCTYPE html> -->
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>Overview</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="css/skeleton.css">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/overview.css">
+  <link rel="stylesheet" href="css/menu.css">
+  <link rel="stylesheet" href="./unfoldable/unfoldable.css">
+  <link rel="stylesheet" type="text/css" src='radial/radial.css'>
+  <link rel="stylesheet" type="text/css" src='discovery/discovery.css'>
+</head>
+<body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/d3/4.2.2/d3.min.js'></script>
+  <script>
+    d3v4 = d3
+    window.d3 = null
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.13/d3.min.js"></script>
+  <script>
+    d3v3 = d3
+    window.d3 = null
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.0.0/lodash.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <script src="./data/data.js"></script>
+  <script src="./switch.js"></script>
+
+  <header><img src="./overview.png"></img></header>
+    <div id="main">
+      <div id="switch">
+        <div class="menu">
+          <div class="btn trigger">
+            <span class="line"></span>
+          </div>
+          <div class="icons">
+            <div class="rotater">
+              <div class="btn btn-icon" id="discovery">
+                <img src="network.svg" class="clicker" ></img>
+              </div>
+            </div>
+            <div class="rotater">
+              <div class="btn btn-icon" id="tree">
+                <img src="plant.svg" class="clicker" ></img>
+              </div>
+            </div>
+            <div class="rotater">
+              <div class="btn btn-icon" id="radial">
+                <img src="network.svg" class="clicker" ></img>
+              </div>
+            </div>
+          </div>
+        </div>
+        <script>
+          $(document).ready(function() {
```

Comment:
if you are only going to use jquery for this one functionality I would avoid importing the whole thing and do it with plain vanilla javascript.

Reply:
Just checked the switch.js and it still feels like it's getting very little use out of jQuery. I think if you modify it to vanilla js you could improve performance and load times.